.env.example

```diff
@@ -0,0 +1,16 @@
+# Adyen API Configuration
```

Comment:
`.env` doesn't apply to the kotlin example, it should be `application.properties`